src/style.css

```diff
@@ -0,0 +1,87 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  margin-top: 20px;
```

Comment:
It is not a good practice to apply such a margin to a body. Better use it at form